applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/explicit_strategy.hpp

```diff
@@ -0,0 +1,537 @@
+//
+//   Project Name:        KratosSolidMechanicsApplication $
+//   Created by:          $Author:           MSantasusana $
+//   Last modified by:    $Co-Author:          JMCarbonel $
```

Comment:
Old header, besides, maybe is agood idea to move this to teh core to avoid code duplication, right @RiccardoRossi